requirements.txt

```diff
@@ -14,7 +14,7 @@ Jinja2>=2.10.1,<2.11
 intelhex>=1.3,<=2.2.1
 mbed-ls>=1.5.1,<1.8
 mbed-host-tests>=1.4.4,<1.6
-mbed-greentea>=0.2.24,<1.7
+mbed-greentea>=0.2.24,<=1.7.1
```

Comment:
Assuming you got latest greentea, this fixes the issue. But shouldn't this be <1.8 rather - if we are already going pass 1.7 version

@bridadan Please review

Reply:
@ARMmbed/mbed-os-tools `mbed-greentea>=0.2.24,<1.8` would be the best here? Locally tested , fixes the error with dependency issue I had today.

Reply:
I agree with you. Just updated as you suggested. Waiting for review from the other people.